docker/native-linux-image.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -x
+set -eo pipefail
+
+# shellcheck disable=SC1091
+. lib.sh
```

Comment:
same here

```suggestion
. /lib.sh
```

Reply:
This happens during the build script, so this example this is fine (since it's always run from root, like `linux-image.sh`, etc). But since we're explicit with `linux-image.sh`, no reason to not be explicit here.

Reply:
I've left this consistent with our other build scripts, like `linux-image.sh` and `common.sh`, so I've kept it as-is.